src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -1017,9 +1019,7 @@ public TFTensor GetBufferedBatchTensor()
         }
     }
 
-    /// <summary>
-    /// Estimator for the <see cref="TensorFlowTransformer"/>.
-    /// </summary>
+    /// <include file='doc.xml' path='doc/members/member[@name="TensorflowTransformer"]/*' />
```

Comment:
>/// [](start = 4, length = 3)

issue #3481  says  the transform should suggest the package  it is part of .  

Has that been added somewhere in this PR ?

Reply:
As a follow up to @abgoswam, why are we not using the usual estimator doc template here?

Reply:
I checked that the existing documentation states that. 

[https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.TensorFlow/doc.xml#L31](https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.TensorFlow/doc.xml#L31)

---
In reply to: [277842926](https://github.com/dotnet/machinelearning/pull/3537#discussion_r277842926) [](ancestors = 277842926)

Reply:
In the interest of time. All the information that the template establishes is already there. The file is referenced from the main package too. 

---
In reply to: [277847407](https://github.com/dotnet/machinelearning/pull/3537#discussion_r277847407) [](ancestors = 277847407)